# I don't know what this should look like, I think it
# probably depends on what MAVROS wants
int8 data